### DISCUSSION TITLE
Preload HLS videos to play instantly

### DISCUSSION THREAD

**Discussion by emptycrown** [cookpete/react-player#1810] [Q&A]
I'm making a page that has a bunch of HLS videos in a scrollable column. I'd like to be able to tell the player to preload the beginning fragment(s) of the streams of the next few videos so that when I scroll to them, they play instantly. 

I can't seem to get this to work with any of the file config params. I saw that support for `preload` was recently removed but that seems unrelated. Am I missing something here? Thanks!

**Comment by gorfadvijay** [comment]
@EmptyCrown Hii there
i want to implement the same thing, have you figured out the solution,if yes then please do let me know

**Comment by jonasinus** [comment]
is there any solution by now? I currently have the same issue
